helmfile/values/prometheus-blackbox-exporter-wc.yaml.gotmpl

```diff
@@ -4,3 +4,42 @@ config:
   modules:
     tcp_connect:
       prober: tcp
+
+serviceMonitor:
+  enabled: true
+  targets:
+    {{- if .Values.prometheusBlackboxExporter.targets.prometheus }}
+    - name: prometheus
+      url: kube-prometheus-stack-prometheus.monitoring.svc.cluster.local:9090
```

Comment:
I think this should have a `/healthz` endpoint. If you check the readiness probe I think you should be able to use the same

Reply:
I like this idea. Will check and see!